packages/eventual-send/src/index.js

```diff
@@ -235,8 +235,36 @@ export function makeHandledPromise() {
         // Sanity checks.
         validateHandler(presenceHandler);
 
+        const { proxy: proxyOpts, resultCallback } = options;
+        let result;
+        let presence;
+        if (proxyOpts) {
+          const {
+            handler: proxyHandler,
+            target: proxyTarget,
+            revocable,
+          } = proxyOpts;
+          if (revocable) {
+            // Create a proxy and its revoke function.
+            result = Proxy.revocable(proxyTarget, proxyHandler);
+            presence = result.proxy;
+          } else {
+            result = new Proxy(proxyTarget, proxyHandler);
+            presence = result;
+          }
+        } else {
+          // Default presence.
+          result = Object.create(null);
+          presence = result;
+        }
+
+        // Provide the resulting proxy (or {proxy, revoke} object).
+        if (resultCallback) {
+          resultCallback(result);
+        }
```

Comment:
```suggestion
        const { proxy: proxyOpts } = options;
        let presence;
        if (proxyOpts) {
          const {
            handler: proxyHandler,
            target: proxyTarget,
            revokerCallback,
          } = proxyOpts;
          if (revokerCallback) {
            // Create a proxy and its revoke function.
            const { proxy, revoke } = Proxy.revocable(proxyTarget, proxyHandler);
            presence = proxy;
            revokerCallback(revoke);
          } else {
            presence = new Proxy(proxyTarget, proxyHandler);
          }
        } else {
          // Default presence.
          presence = Object.create(null);
        }
```

Reply:
As an experiment, I tried the above revision and then did a `yarn test` to find out what tests I'd need to adjust for this API change. All the tests still passed, which indicates that the previous API was not really being tested.

Reply:
This code was mostly adapted from Michaels code from the tc39/eventual-send issue linked in the first comment of this PR.

As someone that prefers implementing revocability oneself it had not occured to me to even test that when I was batttling with how to test presences with proxy. @michaelfig you recall why the resultCallback was needed for other than getting the revoker for revocable?

Reply:
After looking at it and thinking a bit I do not see the any reason not to change resultCallback into revokerCallback as per thine suggestion.